src/components/PostListRow/PostListRow.js

```diff
@@ -43,10 +42,10 @@ const PostListRow = (props) => {
 
   return (
     <div styleName={cx('post-row', { unread, expanded })} onClick={showDetails}>
-      <div styleName='votes'>
+      {/* <div styleName='votes'>
         <a
-          onClick={(e) => {
-            voteOnPost(e)
+          onClick={(e) => { // TODO: all of this needs to be tweaked
```

Comment:
Haven't gotten to PostListRow or Grids views. Do we want to try tackling them now as well???